src/components/AdminConsoleBox.css

```diff
@@ -21,7 +21,7 @@
     display: inline-block;
     width: 35px;
     height: 40px;
-    background: url('/images/admin_icons.png') no-repeat 0 0;
+    background: url('../images/admin_icons.png') no-repeat 0 0;
```

Comment:
CSS url ruleset is handled by a webpack package called css-loader, which is managed and configured by react-scripts. As such, relative links still need to be used within css files.